migrations/fixtures.json

```diff
@@ -0,0 +1,416 @@
+{
+  "DEPLOYER": {
+    "address": "0x8a194c13308326173423119F8dCb785CE14C732B",
+    "type": "external",
+    "shouldHaveEther": true,
+    "seed": "bread sick proud swift orchard wish model mammal brass ready dinner pave runway can twelve best bundle filter stuff sister paddle kangaroo keep supply",
+    "derivationPath": null,
+    "privateKey": "0x2a9f4a59835a4cd455c9dbe463dcdf1b11b937e610d005c6b46300f0fa98d0b1",
+    "notes": ""
+  },
+  "DEPLOYER_UNIVERSE": {
+    "address": "0x30fD2af22459B61F5bdfdDcaeF9BFaD6AcBF9fDC",
+    "type": "external",
+    "shouldHaveEther": true,
+    "seed": "submit defy item boss situate isolate purse major retire nothing mammal usual boil hope sentence group vivid clutch another indoor slam illegal street dust",
+    "derivationPath": "m/44'/60'/0'/0/0",
+    "privateKey": "0x7ccdc091ae2a3cccb338d1aa00f8ec7ff60f30abb9a447a17fe3326ebc0849f1",
+    "notes": "used ONLY to deploy UNIVERSE so this address never changes, do not touch, do not use!"
+  },
+  "INV_ETH_ICBM_NO_KYC": {
+    "address": "0x429123b08DF32b0006fd1F3b0Ef893A8993802f3",
+    "type": "investor",
+    "shouldHaveEther": true,
+    "seed": "argue resemble sustain tattoo know goat parade idea science okay loan float solution used order dune essay achieve illness keen guitar stumble idea strike",
+    "derivationPath": "m/44'/60'/0'/0",
+    "privateKey": "0x79177f5833b64c8fdcc9862f5a779b8ff0e1853bf6e9e4748898d4b6de7e8c93"
+  },
+  "INV_ETH_ICBM_NO_KYC_2": {
+    "address": "0xDE185A5c2Bd3913fAC1F64102e3DEFD9E1797C4d",
+    "type": "investor",
+    "shouldHaveEther": true,
+    "seed": "force squeeze drift rigid dizzy cave random menu gap pudding trip elevator bleak essence decide camp screen glass oppose possible bunker piece merit much",
+    "derivationPath": "m/44'/60'/0'/0/0",
+    "privateKey": "0x26725897cd95531f8f06c963a29424af96ca61684f619316752112702d58e0b0"
+  },
+  "INV_EUR_ICBM_HAS_KYC": {
+    "address": "0xE6Ad2CdBA2FB15504232eBFa82f64c06c87F9326",
+    "type": "investor",
+    "kycClaims": {
```

Comment:
i like it. if that's not too big problem - could we rename it to identityClaims?